src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownMetadataValidationRule.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.MarkdownValidators
+{
+    using System.ComponentModel;
+
+    using Newtonsoft.Json;
+
+    public class MarkdownMetadataValidationRule
```

Comment:
yes, they are same now.

---
In reply to: [73829630](https://github.com/dotnet/docfx/pull/579#discussion_r73829630) [](ancestors = 73829630)